models.py

```diff
@@ -22,10 +17,24 @@ def __init__(self, nom, prenom, date_de_naissance, sexe, classement):
             self.prenom = prenom
             self.date_de_naissance = date_de_naissance
             self.sexe = sexe
-            self.classmeent = classement
+            self.classement = classement
+
+    def __str__(self):
+        return f"{self.nom},{self.prenom},{self.date_de_naissance},{self.sexe},{self.classement}"
+
+
+
+
+class Tours:
+    def __init__(self, add_players, enter_results ):
+        self.add_players = add_players
```

Comment:
replace `add_players` par `players`.
You can add some typing here
players: List[Players] in the signature of `__init__`

Reply:
What is the purpose of enter results there?

Reply:
I think I understand now.